project/src/main/java/com/google/sps/servlets/CompetitionsServlet.java

```diff
@@ -67,24 +69,139 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     }
   }
 
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    String frontEndInfo = getInformationFromFrontEnd(request);
+
+    try {
+      //get specific information from json object
+      JSONObject jsonObj = new JSONObject(frontEndInfo);
+      int userId = jsonObj.getInt("userId");
+      String compName = jsonObj.getString("name");
+      SimpleDateFormat formatter = new SimpleDateFormat("dd-MM-yyyy");
+      Date startDate = formatter.parse(jsonObj.getString("startdate"));
+      Date endDate = formatter.parse(jsonObj.getString("enddate"));
+      JSONArray participants = jsonObj.getJSONArray("list");
+
+      String competitionStmt = String.format(
+          "INSERT INTO competitions (start_date, end_date, competition_name, creator) VALUES "
+              + "(DATE '%tF', DATE '%tF', '%s', %d);",
+          startDate, endDate, compName, userId);
+
+      DataSource pool = (DataSource) request.getServletContext().getAttribute("db-connection-pool");
+      try (Connection conn = pool.getConnection()) {
+
+        //add competition to database
+        addStatementToDataBase(competitionStmt, conn);
+        int competitionId = getLatestInsertedID(conn);
+
+        //add participants to database
+        for (int i = 0; i < participants.length(); i++) {
+          String email = participants.get(i).toString();
+          String findUserStmt = "SELECT name,id FROM users WHERE email=\"" + email + "\";";
```

Comment:
Why not use single quotes?
Also formatting wise, should be a space between name and id